index.html

```diff
@@ -1349,7 +1349,13 @@ <h2>Our Advice for Newbies!</h2>
       <a href="advice/claire-pollock-advice.html">Claire P's Advice</a><br>
       <a href="advice/david-gibson-advice.html" target="_blank">D's Delusions of Wisdom</a><br>
       <a href="advice/jamie-williams-advice.html" target="_blank">Jamie W's Advice</a><br>
+<<<<<<< HEAD
```

Comment:
Hi @jowiekat,

This is great! Just one change to make before we can merge in your code. Can you remove the merge conflict markers on lines 1352, 1354, and 1356? Then add and commit the changes and push your advice branch to GitHub.  Your pull request will update automatically and we should be able to merge in your code!

How does that sound? If you have any questions, let us know!

Best,
Lisa